lib/index.js

```diff
@@ -183,13 +183,18 @@ internals.Client.prototype._request = function (method, url, options, relay, _tr
         uri.ciphers = options.ciphers;
     }
 
+    const start = Date.now();
```

Comment:
I'd like to keep this where it was before so that the timing is only around the request and not the event handler.